dub.sdl

```diff
@@ -29,9 +29,8 @@ subPackage {
     "src/ddmd/tokens.d" \
     "src/ddmd/utf.d"
 
-  // stringImportPaths cannot be used because it will make Dub extremely slow
-  // https://github.com/dlang/dub/issues/1199
-  dflags "-J$PACKAGE_DIR"
+  preGenerateCommands "\"$${DUB_PACKAGE_DIR}config.sh\" \"$${DUB_PACKAGE_DIR}generated/dub\" VERSION /etc"
```

Comment:
It's not documented, but quite accessible to anyone invoking env and using git grep (or just searching the project).
https://github.com/dlang/dub/blob/dafdc47d690c63d074e1b9ed57349fe2eaaabd2b/source/dub/generators/generator.d#L480-L519